cfgov/v1/jinja2/v1/_includes/organisms/well.html

```diff
@@ -0,0 +1,22 @@
+
+{# ==========================================================================
+
+   o_well.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a grey well organism.
+   See [GHE]/flapjack/Modules-V1/wiki/Well
+
+   template: Template URL for content to include.
+
+   ========================================================================== #}
+
+{% macro render(template) %}
+{% import template as content %}
```

Comment:
nvm, I was thinking it should accept both like \:

` if template.indexOf('.html')`

Probably unnecessary though.